lib/npm/tagger.js

```diff
@@ -29,27 +29,48 @@ var Tagger = module.exports = function Tagger(opts) {
  * @returns {undefined}
  */
 Tagger.prototype.delete = function (opts, callback) {
-  var pkg = opts.pkg,
-    tag = opts.tag;
-
-  var update = {
-    name: pkg,
-    distTags: {}
-  };
-
-  //
-  // Note that for maps we actually don't currently delete values as to
-  // prevent tombstones. This should technically be allowed but we should
-  // expect in this case for the value to still exist but have a null value
-  //
-  update.distTags[tag] = null;
+  this._createBody(opts, (error, { update } = {}) => {
+    if (error) return callback(error);
+
+    //
+    // Note that for maps we actually don't currently delete values as to
+    // prevent tombstones. This should technically be allowed but we should
+    // expect in this case for the value to still exist but have a null value
+    //
+    update.distTags[opts.tag] = null;
```

Comment:
do we still want this? With DynamoDB doing overwrites by default, removing `distTags` is relatively easy.

Reply:
@Swaagie we can do this in a way more appropriate for DynamoDB to prevent this null setting. whatever is cleaner for you